benchmark_runner/main/environment_variables.py

```diff
@@ -50,6 +50,9 @@ def __init__(self):
         self._environment_variables_dict['run_types'] = ['test_ci', 'func_ci', 'perf_ci']
         # Run type test_ci/func_ci/perf_ci, default test_ci same environment as func_ci
         self._environment_variables_dict['run_type'] = os.environ.get('RUN_TYPE', 'test_ci')
+        # Benchmark runner local run artifacts path with time stamp format
+        self._environment_variables_dict['time_stamp_format'] = os.path.join(datetime.datetime.fromtimestamp(time.time()).strftime('%Y-%m-%d-%H-%M-%S'))
```

Comment:
I'd really like the timestamp to contain the timezone.  That's going to be needed to make sense of the prometheus snapshot.